<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SearchLineEdit">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mediumTab">
       <attribute name="title">
        <string>medium</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="mediumTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>83</number>
            </attribute>
            <column>
             <property name="text">
              <string>type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>id</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>title</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>author</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>duration / pages</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>quality</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>year</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>given to</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="deleteMediumButton">
              <property name="text">
               <string>delete selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveMediumButton">
              <property name="text">
               <string>change selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="giveButton">
              <property name="text">
               <string>give selected to ...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="returnButton">
              <property name="text">
               <string>return selected</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="personTab">
       <attribute name="title">
        <string>person</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="personTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <column>
             <property name="text">
              <string>id</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>birthday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>medium</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="personButtonsLayout">
            <item>
             <widget class="QPushButton" name="deletePersonButton">
              <property name="text">
               <string>delete selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="savePersonButton">
              <property name="text">
               <string>change selected</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>fi&amp;le</string>
    </property>
    <addaction name="actionRead_from"/>
    <addaction name="actionWrite_to"/>
   </widget>
   <widget class="QMenu" name="menunew">
    <property name="title">
     <string>add &amp;new...</string>
    </property>
    <addaction name="actionPerson"/>
    <addaction name="separator"/>
    <addaction name="actionMedium"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menunew"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRead_from">
   <property name="text">
    <string>&amp;Read from</string>
   </property>
  </action>
  <action name="actionWrite_to">
   <property name="text">
    <string>&amp;Write to</string>
   </property>
  </action>
  <action name="actionPerson">
   <property name="text">
    <string>&amp;Person</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>&amp;Medium</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
